README.md

```diff
@@ -150,12 +150,12 @@ cargo run --bin ln_gateway --release cfg
 ```
 To get an invoice you can use ln2 :
 ```shell
-#make sure your client holds coins for that amount tier. 100000msat = 100sat. For now the client is not able to spend with change
-lightning-cli --network regtest --lightning-dir=ln2 invoice 100000 test test 1m
+#Make sure your peg-in created coins of small enough tiers
```

Comment:
The remark about change could have stayed imo. Maybe I should have put it there in my comment instead of only "…" :sweat_smile: